config/checkstyle/checkstyle.xml

```diff
@@ -177,7 +177,7 @@
                      value="Type name ''{0}'' must match pattern ''{1}''."/>
         </module>
         <module name="MemberName">
-            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9]*$"/>
+            <property name="format" value="^[a-z][a-z0-9]*(_[a-zA-Z0-9]+)*$"/>
```

Comment:
Add rule to avoid names with  _ : example full_Name, needed to extract response in a class with this element